shared/js/dataDriver.js

```diff
@@ -149,6 +151,12 @@ loop.DataDriver = function() {
             console.log(`Error handling EventSource.put: ${event.data}`, ex);
           }
           break;
+        case "error":
+          setTimeout(
+            this._connectToRoom.bind(this, this._roomToken),
```

Comment:
Yes, after the connection has been opened, the error handler will be called for a connection which is closed.

I tested by unplugging my ethernet cable.